### DISCUSSION TITLE
pysu2 not found, openBLAS not found, Apple clang instead of gcc

### DISCUSSION THREAD

**Discussion by bakakouhai** [su2code/SU2#1253] [Q&A]
Good morning.

I successfully compiled SU2 with:
a. autodiff = true
b. directdiff = true
c. pywrapper = true
d. enable-tests = true.

However, when I finished the serial regression test, all pywrapper test have failed.
Running flatPlate_rigidMotion case via launch_flatPlate_rigidMotion.py returns:
    ModuleNotFoundError: No module named '_pysu2'

My Desktop is as follows:
Macos BigSur 11.2.1
Apple clang version 12.0.0 (clang-1200.0.32.29)
gcc [homebrew]
OpenMPI 4.1.0 [homebrew]
SU2 7.1.1 Blackbird
Python 3.9.2 [homebrew]

in the bash.profile I have these lines:
default to python3 homebrew
    alias python="python3"
    alias pip="pip3"

default to homebrew gcc
    alias gcc="gcc-10"
    alias cc="gcc-10"
    alias g++="g++-10"
    alias c++="c++-10"
    alias gfortran="gfortran-10"

SU2
    export SU2_RUN=/usr/local/bin
    export SU2_HOME=/Users/iwan/SU2
    export PATH=$PATH:$SU2_RUN
    export PYTHONPATH=$PYTHONPATH:$SU2_RUN

Other issues that I notice:
1. OpenBLAS cannot be found [installed via homebrew] if I set -Denable-openblas=true
2. The compiler used was the Apple clang even after I use aliases to gcc in bash.profile

Is there anything wrong with my setting that cause the module cannot be found?
I have SU2 folder containing the python script in usr/local/bin/SU2.
I also have SU2_CFD.py in usr/local/bin

Thank you.

**Comment by pcarruscag** [comment]
Hi, not sure what might be the problem with python.
Have you tried the common solution for blas problems described in https://su2code.github.io/docs_v7/Build-SU2-Linux-MacOS/#linear-algebra-options ?

**Reply by bakakouhai** [reply]
Thank you for the fast response.

I installed lapack and rerun with -Denable-openblas=true and -Dextra-deps=lapack.

The result is the same: cannot find dependency: openblas.

Attached is the meson.log for configuring with:
./meson.py build -Denable-autodiff=true -Denable-directdiff=true -Denable-pywrapper=true -Denable-tests=true -Denable-openblas=true -Dextra-deps=lapack --reconfigure

Thank you.

[meson-log.txt](https://github.com/su2code/SU2/files/6264794/meson-log.txt)

**Reply by patelha57** [reply]
Hi @bakakouhai , did you find a solution to this issue? I'm experiencing a similar error
